pkg/identityserver/user_registry_test.go

```diff
@@ -54,8 +54,8 @@ func TestValidatePasswordStrength(t *testing.T) {
 			"password1":    false, // Too common.
 			"āaaAAA123!@#": true,
 			"       1A":    true,
-			"āaa	AAA123 ": true,
-			"āaaAAA123 ": true,
+			"āaa	AAA123 ":  true,
+			"āaaAAA123 ":   true,
```

Comment:
Doesn't look like the right spacing.
```suggestion
			"āaaAAA123 ":    true,
```

Reply:
?

Reply:
Was gonna approve and write the same, my question would be if its a mistake on the part of `go:fmt` or not.

Reply:
This is what `gofumpt` prefers - I didn't format this manually. I suppose does not match our intuition because the character spacing is weird for these esoteric letters.
<img width="443" alt="image" src="https://user-images.githubusercontent.com/36161392/192557155-e5495b10-6c9b-4325-8b83-0889d5bb28e3.png">